documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -67,12 +71,49 @@ from additional images.
 This section describes a typical auxiliary image configuration for the
 Model in Image use case.
 
+#### Auxiliary volumes and paths
+
+A domain resource `domain.spec.auxiliaryImageVolumes`
+auxiliary image volume defines a `mountPath`, `name`, plus
+[additional optional fields](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md#auxiliary-image-volume).
+The `mountPath` field is the location of a directory in an auxiliary image, and
+is also the location in the main pod container (which will automatically contain
+a recursive copy of the auxiliary image directory). The `name` field is 
+arbitrary, and is in turn referenced by one or more auxiliary images that 
+are defined separately using
+[Auxiliary images](#auxiliary-images) configuration.
+
+For example:
+
+```
+  spec:
+    auxiliaryImageVolumes:
+    - name: auxiliaryImageVolume1
+      mountPath: /auxiliary
+```
+
 #### Auxiliary images
 
-One or more auxiliary images can be configured on a domain resource `serverPod`.
+One or more auxiliary images can be configured in a domain resource
+`serverPod.auxiliaryImage` array.
+Each array entry must define an `image` and `volume`
+where `image` is the name of an auxiliary image
+and the `volume` is the name of an [auxiliary image volume](#auxiliary-volumes-and-paths)
+as described above.
+You can optionally also specify an `imagePullPolicy`,
+which defaults to `Always` if the `image` ends in `:latest` and to `IfNotPresent`,
+otherwise.
+You can also optionally customize
+the command that is used to merge (copy) the auxiliary image's files
+into the auxiliary image volume during pod startup (this is rarely 
+needed, see [Performing replaces instead of merges](#performing-replaces-instead-of-merges) below for an example).
```

Comment:
see [Performing replaces instead of merges](#performing-replaces-instead-of-merges) below for an example). -> see [Performing replaces instead of merges](#performing-replaces-instead-of-merges) for an example).

**below** (adv) (prep) Do not use to refer to material that can be found at a later position in a document.